cmd/kubeapps-apis/plugins/fluxv2/packages/v1alpha1/repo.go

```diff
@@ -514,11 +526,12 @@ func (s *Server) setOwnerReferencesForRepoSecret(
 func (s *Server) updateKubeappsManagedRepoSecret(
 	ctx context.Context,
 	repoName types.NamespacedName,
+	typ string,
```

Comment:
Just a slight preference towards `type` instead of `typ`, but up to you.

Reply:
yeah I wanted to do that. But VSCode is not letting me. I think 'type' may be a reserved keyword ?